pkg/gardenlet/controller/managedseed/charttest/charttest.go

```diff
@@ -1030,9 +1027,17 @@ func ComputeExpectedGardenletDeploymentSpec(
 			}
 
 			if value, ok := deploymentConfiguration.Resources.Limits[corev1.ResourceCPU]; ok {
+				if deployment.Template.Spec.Containers[0].Resources.Limits == nil {
+					deployment.Template.Spec.Containers[0].Resources.Limits =
+						map[corev1.ResourceName]resource.Quantity{}
```

Comment:
No need to put `map[corev1.ResourceName]resource.Quantity{}` to a new line.

```suggestion
					deployment.Template.Spec.Containers[0].Resources.Limits = map[corev1.ResourceName]resource.Quantity{}
```